src/main.cpp

```diff
@@ -0,0 +1,1605 @@
+#include <algorithm>
+#include <chrono>
+#include <fstream>
+#include <limits>
+#include <thread>
+#include <vector>
+
+#include "Log.hpp"
+
+#ifdef __linux__
+#include <vulkan/vulkan.h>
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_vulkan.h>
+#endif
+
+#ifdef _WIN32
```

Comment:
There is no build support for Windows, so please remove code flagged for Windows.